Gruntfile.js

```diff
@@ -1,10 +1,24 @@
+'use strict';
+
 module.exports = function(grunt) {
-  'use strict';
+  require('load-grunt-tasks')(grunt);
   grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
     eslint: {
       src: ["lib/**/*.js"]
+    },
+    shell: {
```

Comment:
Might want to use https://github.com/kmiyashiro/grunt-mocha instead of shelling out.